Resources/Storage/StorageFactory.py

```diff
@@ -89,7 +89,7 @@ def getStorages( self, storageName, pluginList = None ):
     if pluginList is None:
       pluginList = []
     if not self.vo:
-      return S_ERROR( 'Mandatory vo parameter is not defined' )
+      gLogger.warn( 'No VO information available' )
```

Comment:
I see... For us it does not change anything anyway, however you are right, one should think more in depth about the multi VO paradigm. But maybe, one should do it before doing small patches here and there ?